### DISCUSSION TITLE
Why are we using vanilla css ?

### DISCUSSION THREAD

**Discussion by tuxy-streamer** [RajeshTechForge/pinac_workspace#87] [Q&A] [answered]
We could use any of the faster alternatives to css.
Why are we a adding bunch of css files in every where?

**Comment by RajeshTechForge** [comment, marked as answer]
As a developer, I prefer using Vanilla CSS in my projects for its simplicity and easy for everyone. In terms of performance, it is more than sufficient.

When it comes to adding numerous CSS files throughout the project, I've restructured the files multiple times and discovered that breaking down the files for each function makes it much easier to _maintain_, _change anything_, and is also _easy for contributors to work on_.

**Reply by tuxy-streamer** [reply]
I didn't meant performance when I said faster alternative.
By faster I meant faster for development.
But it's fine if you're up to simplicity.
